app/controllers/users/noticed_notifications_controller.rb

```diff
@@ -11,8 +11,10 @@ def index
   def mark_as_read
     notification = NoticedNotification.find(params[:notification_id])
     notification.update(read_at: Time.zone.now)
-    project = notification.params[:project]
-    redirect_to user_project_path(project.author, project)
+    answer = NotifyUser.new(params).call
+    return redirect_to group_assignment_path(answer.first_param, answer.second) if answer.type == "new_assignment"
```

Comment:
1. Why introduce a new `type` in service when we can already compare with schema `type`.
2. Stick to `if-else` block as we have some other notification coming up which may have a different path other than `project` or `group`. 
Else LGTM @tachyons WDYT?